test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -44,6 +45,40 @@ public async Task GetWorkerStatusesAsync_ReturnsExpectedResult()
             }
         }
 
+        [Fact]
+        public async Task WorkersStartInLinearTime()
+        {
+            TimeSpan startupInterval = TimeSpan.FromSeconds(2);
+            int expectedProcessCount = 4;
+            RpcFunctionInvocationDispatcher functionDispatcher = GetTestFunctionDispatcher(expectedProcessCount, processStartupInterval: startupInterval);
+
+            Stopwatch stopwatch = Stopwatch.StartNew();
+            await functionDispatcher.InitializeAsync(GetTestFunctionsList(RpcWorkerConstants.NodeLanguageWorkerName));
+
+            await TestHelpers.Await(() =>
+            {
+                var currentChannelCount = functionDispatcher.JobHostLanguageWorkerChannelManager.GetChannels().Count();
```

Comment:
nit: Seeing this in a few different spots. Can you please update to the actual type instead of `var` on method return assignments?